packages/vm/test/api/EIPs/eip-6780-selfdestruct-same-tx.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import { Transaction } from '@ethereumjs/tx'
+import { Account, Address, privateToAddress } from '@ethereumjs/util'
+import { equalsBytes, hexToBytes } from 'ethereum-cryptography/utils'
+import * as tape from 'tape'
+
+import { VM } from '../../../src/vm'
+const pkey = hexToBytes('20'.repeat(32))
+
+const GWEI = BigInt(1000000000)
+const sender = new Address(privateToAddress(pkey))
+
+const common = new Common({
+  chain: Chain.Mainnet,
+  hardfork: Hardfork.London,
+  eips: [6780],
+})
+
+const payload = hexToBytes('60016001556001FF')
```

Comment:
A comment like "Payload includes `SELFDESTRUCT` `FF` opcode" above this line would make it more clear that the instruction is being executed as part of the transaction.

Reply:
Have addressed!